suite/tests/CMakeLists.txt

```diff
@@ -52,7 +52,7 @@ cmake_minimum_required(VERSION 3.7)
 include(../../make/policies.cmake NO_POLICY_SCOPE)
 include(../runsuite_common_pre.cmake) # For _DR_set_VS_bitwidth_env_vars.
 
-option(TEST_LONG "run long set of tests")
+option(RUN_LONG_SUITE "run long set of tests")
```

Comment:
I don't think `TEST_LONG` was documented anywhere. I'll add something for `RUN_LONG_SUITE` once this PR is submitted.